### DISCUSSION TITLE
responseBodyTest for a boolean response

### DISCUSSION THREAD

**Discussion by jpjpjp** [apideck-libraries/portman#670] [Q&A]
I have an API which returns a boolean as a response (simply true or false)

Is there a way to write a responseBodyTest for this?  Does it seem like the syntax requires the `key` key and then another key (i.e., `value`).   Perhaps there is a way to set `key` to the response body itself instead of a property within a JSON object?

**Comment by thim81** [comment]
hi @jpjpjp 

This is untested, but typically you can use 

```
key: .
value: true
```

> key (String) : The key that will be targeted in the request body to overwrite/extend. Use the . notation to target nested properties. To target the root level, use . as key.

**Reply by jpjpjp** [reply]
Works like a charm @thim81 !

I submitted https://github.com/apideck-libraries/portman/pull/671 to update the docs with this info.
